test/ttt_test.exs

```diff
@@ -78,47 +78,47 @@ defmodule TTTTest do
   end
 
   test "Player tokens are updated and rendered on the board when user provides valid input" do
-    board = Board.setup
-    p1 = Player.human("Steven", :x)
-    p2 = Player.human("Connie", :o)
-    game = Game.setup(p1, p2)
-    {:ok, game} = Game.play_turn(game, :o, {0, 0})
-    {:ok, game} = Game.play_turn(game, :x, {2, 1})
-    {:ok, game} = Game.play_turn(game, :o, {1, 0})
-    {:ok, game} = Game.play_turn(game, :x, {1, 1})
-    {:ok, game} = Game.play_turn(game, :o, {2, 0})
+    board = Board.setup("", "")
+    p1 = Player.human({"Steven", "x"})
+    p2 = Player.human({"Connie", "o"})
+    game = Game.setup(p1, p2, 1)
+    {:ok, game} = Game.play_turn(game, p2, {0, 0})
+    {:ok, game} = Game.play_turn(game, p1, {2, 1})
+    {:ok, game} = Game.play_turn(game, p2, {1, 0})
+    {:ok, game} = Game.play_turn(game, p1, {1, 1})
+    {:ok, game} = Game.play_turn(game, p2, {2, 0})
 
     assert capture_io(fn -> TTT.update_visual(board, game) end) ==
-      "\n\n o | o | o\n---+---+---\n 4 | x | x\n---+---+---\n 7 | 8 | 9\n\n\n"
+      "\n\n o | o | o\n-----------\n 4 | x | x\n-----------\n 7 | 8 | 9\n\n\n"
   end
 
   test "Naive computer player will always take the next open spot following opponent's move if there are no preceding spots" do
-    p1 = Player.human("Steven", :x)
-    p2 = Player.naive_computer("Comp", :o)
-    game = Game.setup(p1, p2)
+    p1 = Player.human({"Steven", :x})
+    p2 = Player.naive_computer({"Comp", :o})
+    game = Game.setup(p1, p2, 1)
 
-    {:ok, game} = Game.play_turn(game, :x, {0, 0})
+    {:ok, game} = Game.play_turn(game, p1, {0, 0})
 
     assert TTT.generate_naive_move(game, 1) == {1, 0}
   end
 
   test "Naive computer player will always take the first available spot preceding the opponent's move" do
-    p1 = Player.human("Steven", :x)
-    p2 = Player.naive_computer("Comp", :o)
-    game = Game.setup(p1, p2)
+    p1 = Player.human({"Steven", :x})
+    p2 = Player.naive_computer({"Comp", :o})
+    game = Game.setup(p1, p2, 1)
 
-    {:ok, game} = Game.play_turn(game, :x, {2, 0})
+    {:ok, game} = Game.play_turn(game, p1, {2, 0})
 
     assert TTT.generate_naive_move(game, 1) == {0, 0}
   end
 
   test "Random computer will generate random moves" do
     :rand.seed(:exs1024, {123, 123_534, 345_345})
-    p1 = Player.human("Steven", :x)
-    p2 = Player.random_computer("Comp", :o)
-    game = Game.setup(p1, p2)
+    p1 = Player.human({"Steven", :x})
+    p2 = Player.random_computer({"Comp", :o})
```

Comment:
Try to use `player_one` and `player_two` instead of `p1` and `p2`. Good names make your code easier to read!